src/components/base/outline-google-map/src/map-selector.ts

```diff
@@ -0,0 +1,137 @@
+import { LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { XSelection } from './selection';
+
+@customElement('map-selector')
+export class MapSelector extends LitElement {
+  @property({ type: String })
+  activateEvent = 'tap';
+
+  @property({ type: String })
+  selectedAttribute: string | null = null;
+
+  @property({ type: Number, reflect: true })
+  selected: number | string | null = null;
+
+  _selection: XSelection<Node> = new XSelection((item, isSelected) =>
+    this.applySelection(item, isSelected)
+  );
+
+  _items: Array<Node> = [];
+
+  get items(): Array<Node> {
+    return this._items;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.addEventListener('slotchange', event => {
```

Comment:
Does this work? The `slotchange` occurs on slots, not components or slotted elements. So, I don't think this will ever fire.

I may be wrong.

If you want to watch for changes, I think you need to add this to a slot or add a mutation observer to the element.